src/inference/src/model_reader.cpp

```diff
@@ -118,6 +118,11 @@ std::shared_ptr<ov::Model> read_model(const std::string& modelPath,
     std::string model_path = modelPath;
 #endif
 
+#if defined(__linux__) && !defined(__ANDROID__)
```

Comment:
Does import_model also need check symlink?

Reply:
The import_model() parameter is std::istream not a file, I think no need to check

Reply:
Why is check only for linux not for other OS. 
If model read directly by frontend the symlink will be not checked.

Reply:
@praasz What u mean about "model read directly by frontend"? The input model path like *.onnx, *.tf not *.xml?